portfolio/src/main/webapp/script.js

```diff
@@ -39,66 +39,47 @@ function deleteComment(key){
+function createMarkerWindow(marker, message) {
+    let newInfoWindow = new google.maps.InfoWindow({
+        content: message
     });
+    newInfoWindow.open(map, marker);
+} 
 
-    let homeMarker = new google.maps.Marker({
-        position: {lat: 34.049, lng: -111.094},
+function createStaticMarker(lat_coord, lng_coord, map, heading) {
+    let staticMarker = new google.maps.Marker({
+        position: {lat: lat_coord, lng: lng_coord},
         map: map,
-        title: "My Home State"
-    });
-
-    homeMarker.addListener('click', function() {
-        homeMarkerInfo.open(map, homeMarker);
-    });
-
-    let travelMarkerInfo = new google.maps.InfoWindow({
-        content: "<h1> Italy </h1> " + "</br>" + "<p> If I could go anywhere in the world, I would go to Italy. I have never been there before </p>"
-    })
-
-    let travelMarker = new google.maps.Marker({
-        position: {lat: 41.872, lng: 12.567},
-        map:map,
+        title: heading,
     });
+    return staticMarker;
+}
 
-    travelMarker.addListener('click', function(){
-        travelMarkerInfo.open(map, travelMarker);
-    });
+let map=null;
+function createMap() {
+    map = new google.maps.Map(
+    document.getElementById('map'),
+    {center: {lat: 37.422, lng: -122.084}, zoom: 4, mapTypeId: 'hybrid'});
 
-    firstTimeGoogleInfo = new google.maps.InfoWindow({
-        content: "<h6> Seattle </h6>" + "</br>" + "<p> My first experience being at Google was here in Seattle the summer of 2019!</p>"
-    });
+    let homeMarker = createStaticMarker(34.049, -111.094, map);
+    homeMarker.addListener('click', function() {createMarkerWindow(homeMarker, "<h6> Phoenix </h6>" + "</br>" + "<p>I have grown up in Arizona all my life. I lived in Virgina for three months when I was eight because my dad is from there but otherwise I have only lived in AZ</p>", "My Home State!")});
 
-    let firstTimeGoogle = new google.maps.Marker({
-        position: {lat: 47.606, lng: -122.332},
-        map:map,
-    });
+    let travelMarker = createStaticMarker(41.872, 12.567, map);
+    travelMarker.addListener('click', function(){createMarkerWindow(travelMarker, "<h6> Italy </h6> " + "</br>" + "<p> If I could go anywhere in the world, I would go to Italy. I have never been there before </p>")});
 
-    firstTimeGoogle.addListener('click', function(){
-        firstTimeGoogleInfo.open(map, firstTimeGoogle);
-    });
+    let firstTimeGoogle = createStaticMarker(47.606, -122.332, map);
+    firstTimeGoogle.addListener('click', function() {createMarkerWindow(firstTimeGoogle, "<h6> Seattle </h6>" + "</br>" + "<p> My first experience being at Google was here in Seattle the summer of 2019!</p>")});
 }
 
 let places_been = [["37", "-76"],["33","-118"], ["34","-118"]];
 function createMarkers() {
     let lat_guess = document.getElementById("lat").value;
     let lng_guess = document.getElementById("lng").value;
-    let temp ="";
+    let temp = "";
```

Comment:
previous comment about changing a variable to "answer" or something similar was in regards to the variable "temp" although it looks like that wasn't obvious based on the way github highlights code when leaving a comment.